runner/operator/access/v1_0_0/snps_and_indels/test/fixtures/files_metadata.json

```diff
@@ -0,0 +1,249 @@
+[
+    {
+        "model": "file_system.filemetadata",
+        "pk": "fffff52a-0696-48d4-9ca5-6580daa44ab2",
+        "fields": {
+            "latest": true,
+            "created_date": "2020-01-06T19:59:50.583Z",
+            "modified_date": "2020-01-06T19:59:50.583Z",
+            "file": "fffff52a-0696-48d4-9ca5-6580daa44ab3",
+            "version": 0,
+            "metadata": {
+                "R": "R1",
```

Comment:
Lets see if we can get away with only having the metadata that we need for the test here, to make this pr smaller